core/tests/protractor/staticPagesTour.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview  End-to-end tests which visits the static pages
+ * and check for any console errors
+ */
+
+var general = require('../protractor_utils/general.js');
+
+describe('Oppia static pages tour', function() {
+  beforeEach(function() {
+    browser.get(general.SERVER_URL_PREFIX);
+  });
+
+  it('visits the links in About dropdown', function() {
+    var dropdown = element(by.css('.protractor-test-about-oppia-list-item'));
+    var linkClassNames = ['.protractor-test-about-link',
+                          '.protractor-test-teach-link',
+                          '.protractor-test-contact-link'
+                         ];
+
+    linkClassNames.forEach(function(className) {
+      browser.actions().mouseMove(dropdown).perform();
+      dropdown.element(by.css(className)).click();
+    });
+  });
+
+  it('visits the donate link', function() {
+    element(by.linkText('Donate')).click();
```

Comment:
No, we don't want to use linkText for anything. Use classes. See guidance [here](https://github.com/oppia/oppia/wiki/Writing-End-to-End-Tests#writing-utilities).